.eslintrc.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
   plugins: ['markdown', 'cypress', 'react-hooks'],
   globals: {
     __DEV__: true,
+    Badge: true, // global MDX component
```

Comment:
I think it should be added at `md/mdx` section to affect these files only.

Reply:
You're right, I've updated it.